### PR TITLE
[UI-15] Add examples for existing components

### PR DESCRIPTION
**Pull request by maciek15**
#### Task

- **Issue or task referred:** UI-15
- **Feature or Bugfix:** Feature

I've added more Storybook examples to components listed below. I did not apply any changes to components that will be refactored: Breadcrumbs, Popover, Tooltip. Additionally I've made some changes to some components styling.

#### Components

- Button (minimal margin to both sides)
- Badge (padding and default colors added)
- Checkbox
- List (default bullet added -> '-')
- Modal
- Notification (default style added)
- NumberInput
- Pill(default styling added)
- Radio (sizes added: 'small', large')
- Switcher
- TextInput

### CODE REVIEW

**Comment by rangoo94** [comment]
If component requires other component only for Storybook, it should be included in `devDependencies` section

File: packages/badge/stories.js [hunk 1]
```diff
@@ -1,11 +1,52 @@
...
+    <Badge>1</Badge>{' '}
+    <Badge>10</Badge>{' '}
+    <Badge>100</Badge>{' '}
+    <Badge>Badge</Badge>
+  </div>
+)
+addStory('inside button', readme, () =>
+  <div>
+    <Button>
+      Mail <Badge>10 unreaded</Badge>
```
Comment (michalochman) [inline packages/badge/stories.js]: unread ;)

File: packages/modal/stories.js [hunk 2]
```diff
@@ -32,5 +33,9 @@ const getInitialState = () => {
     isOpen: false
   }
 }
+const options = {
+  propTypes: [ Modal, Button ],
+  propTablesExclude: [ Button ]
```
Comment (michalochman) [inline packages/modal/stories.js]: Are you sure we need `Button` in both?

Comment (maciek15) [reply packages/modal/stories.js]: Yes, it was required to display headers for PropTypes table of modal. But this problem is fixed in #45 so I will delete it now

**Pull request merged**